### ISSUE TITLE
RfcFunction only works with default mapper

### ISSUE THREAD

**Issue by Noplace** [nunomaia/NwRfcNet#34]
Hi, if I set a mapper through the RfcFunction.Mapper Property it doesn't consider it, the RFC only worked when I modified the default mapper.

**Comment by nunomaia** [comment]
what version of NwRfcNet are you using ?

**Comment by Noplace** [comment]
The latest version available in nuget visual studio, I tried further it
works if I chance the connection mapper property.

Great library looking forward to using in in production

**Comment by nunomaia** [comment]
This is still an issue ?

**Comment by Noplace** [comment]
yes, the version used from Nuget hasn't been updated , still 0.5.0
